app/routes/resources.manifest[.]webmanifest.ts

```diff
@@ -0,0 +1,73 @@
+import { json } from '@remix-run/node';

```

Comment:
provavelmente vc saca mais de PWA que eu que nunca mexi, então aqui to imaginando que vc ja tem essa xp, por isso nao tem mt oq revisar, só queria saber se esse foi o jeito que voce encontrou no remix pra fazere isso, digo o lance da rota e tal

Reply:
yep. running like a charme. Just hit "add to home screen and "voa lá"!